resource/cassandra-reaper-cassandra.yaml

```diff
@@ -44,6 +48,6 @@ server:
     appenders: []
 
 cassandra:
-  clusterName: "test"
-  contactPoints: ["127.0.0.1"]
+  clusterName: "reaper"
+  contactPoints: ["cassandra"]
```

Comment:
I'd like to keep 127.0.0.1 here as this yaml file is a sample that works right away if you have a local cassandra instance (ccm or regular install). 
Can you modify it on the fly when building the container ?